OpenRA.Game/Support/Benchmark.cs

```diff
@@ -46,7 +46,7 @@ public void Write()
 			foreach (var sample in samples)
 			{
 				var name = sample.Key;
-				Log.AddChannel(name, "{0}{1}.csv".F(prefix, name));
+				Log.AddChannel(name, "{0}{1}.csv".F(prefix, name), true);
```

Comment:
Sorry for the confusion! But this should not be added. Leave it at default false.

Reply:
No worries, all taken care of.